### PR TITLE
Heads of staff (and qm) get a box of radio keys for their department

### PR DESCRIPTION
**Pull request by etherware-novice**
## About The Pull Request
All heads, captain, and qm get a box of radio keys for their own departments. Captain's is unique and gets a box of keys for each head, a command key, and an amp key.

Clowns get a box of fake heads of staff keys as well, to prank people with cause why not

## Why It's Good For The Game
Heads (and QM!) dont have much control over who has their radio. Currently, the ONLY ways to get radio keys is by buying the associated radio headset from a vender, and qm can buy the mining conscript.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

radiobox

</details>

## Changelog
:cl:
add: Heads and QM get spare radio key boxes
add: Clown has ordered some fake encryption keys for their pranks
/:cl:

### CODE REVIEW

**Comment by itsmeow** [comment]
Maybe combine this with the amplification keys box. Having so many boxes in the locker is not fun

**Comment by itsmeow** [comment]
add clown fake radio keys to the CL

**Review by Penwin0 — APPROVED** [review]

File: code/game/objects/items/devices/radio/encryptionkey.dm [hunk 1]
```diff
@@ -37,9 +37,13 @@
 
 /obj/item/encryptionkey/amplification
 	name = "amplification module key"
-	desc = "An amplification module key for a radio headset. It will enable the \"Loud mode\" ability on any headset it is inserted into."
 	amplification = TRUE
 
+// Makes sure neither channel messages show up.
+/obj/item/encryptionkey/amplification/Initialize()
+	. = ..()
+	desc = "An amplification module key for a radio headset. It will enable the \"Loud mode\" ability on any headset it is inserted into."
```
Comment (PowerfulBacon) [inline code/game/objects/items/devices/radio/encryptionkey.dm:45]: This feels dodgy

Comment (etherware-novice) [reply code/game/objects/items/devices/radio/encryptionkey.dm:45]: wdym

Comment (Penwin0) [reply code/game/objects/items/devices/radio/encryptionkey.dm:45]: It does but otherwise the parent proc botches up the description

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**